go/api/generic_commands.go

```diff
@@ -312,6 +314,176 @@ type GenericBaseCommands interface {
 	// [valkey.io]: https://valkey.io/commands/pttl/
 	PTTL(key string) (Result[int64], error)
 
+	// Sorts the elements in the list, set, or sorted set at key and returns the result.
+	// The sort command can be used to sort elements based on different criteria and apply
+	// transformations on sorted elements.
+	// To store the result into a new key, see the sortStore function.
+	//
+	// Parameters:
+	// key - The key of the list, set, or sorted set to be sorted.
+	//
+	// Return value:
+	// An Array of sorted elements.
+	//
+	// Example:
+	//
+	// result, err := client.Sort("key")
+	// result.Value(): [{1 false} {2 false} {3 false}]
+	// result.IsNil(): false
+	//
+	// [valkey.io]: https://valkey.io/commands/sort/
+	Sort(key string) ([]Result[string], error)
+
+	// Sorts the elements in the list, set, or sorted set at key and returns the result.
+	// The sort command can be used to sort elements based on different criteria and apply
+	// transformations on sorted elements.
+	// To store the result into a new key, see the sortStore function.
```

Comment:
```suggestion
	// To store the result into a new key, see the SortStore function.
```
Is there a way do add a link to another method in go docs?